src/app/components/BookFilters/BookFilters.jsx

```diff
@@ -37,7 +35,8 @@ class BookFilters extends React.Component {
   }
 
   componentWillReceiveProps(nextProps) {
-    this.setState({ selectedFilters: nextProps.selectedFilters });
+    const newFilters = utils.getFilters(nextProps.filters);
```

Comment:
You can rename this variable to be `filters`. For creating objects, you can use the shortcut:
```
{
  selectedFilters: nextProps.selectedFilters,
  filters,
}
```

In the example above, the key is `filters` but since there's a variable with the same name, you do not have to do `filters: filters`.

Reply:
or you can just do `filters: utils.getFilters(nextProps.filters)` if you don't need to assign `utils.getFilters(nextProps.filters)` anywhere else.